mos_tests/murano/test_cli/murano_test.py

```diff
@@ -0,0 +1,163 @@
+           'OS_USERNAME="{0}" OS_PASSWORD="{1}" ').format(
+            user, password, project)
+    return env
+
+
+def execute_on_all_controllers(env, cmd, fail_ok=False):
+    controllers = env.get_nodes_by_role('controller')
+    all_output = []
+    for controller in controllers:
+        with controller.ssh() as remote:
+            result = remote.execute(cmd)
+            if not fail_ok and not result.is_ok:
+                raise exceptions.CommandFailed(result['exit_code'],
+                                               cmd,
+                                               result.stdout_string,
+                                               result.stderr_string)
+            all_output.append(result)
+    return all_output
+
+
+def wa_for_1543135(env):
+    # WA for "Wrong OS_AUTH_URL makes keystone operations fail"
+    # https://bugs.launchpad.net/mos/+bug/1543135
+    # Changes OS_AUTH_URL from ":5000/" --> ":5000/v2.0" in /root/openrc
+    cmd = unicode(r"sed -i 's#:5000/\x27#:5000/v2.0\x27#g' /root/openrc")
+    execute_on_all_controllers(env, cmd)
+
+
+@pytest.mark.testrail_id('543127')
+@pytest.mark.testrail_id('543125')
+def test_user_with_permissions_can_share_pkg(
+        env, openstack_client, cli_to_controller, restore_murano_policy):
+    """Tests modifying Murano pkg as 'service' project user.
+    QA-1572 [Murano Features] User with all permissions can set
+        'shared' package to 'unshared'.
+    QA-1570 [Murano Features] Sharing Murano App with all permissions.
+    Actions:
+    1. Create new role 'can_publicize_packages';
+    2. Create new user inside 'services' project;
+    3. Assign new role to the new user;
+    4. On all controllers add new role to /etc/murano/policy.json:
+        from:
+        "publicize_package": "rule:admin_api",
+        to:
+        "publicize_package": "rule:admin_api or role:can_publicize_packages",
+    5. As a new user import Murano pkg;
+    6. As a new user update imported pkg with Public=TRUE;
+    7. Check that pkg became public.
+    8. As a new user update imported pkg with Public=FALSE;
+    9. Check that pkg is not public.
+
+    !!! At the moment of writing this test-case, MOS has several bugs:
+    !!!    https://bugs.launchpad.net/mos/+bug/1551575
+    !!!    https://bugs.launchpad.net/mos/+bug/1543135
+    """
+    pkg_for_import = 'io.murano.apps.docker.Interfaces'
+    murano_policy_file = '/etc/murano/policy.json'
+    role_name = 'can_publicize_packages'
+    new_user = {'name': '_test_user_{0}'.format(rand_num()),
+                'password': 'password',
+                'project': 'services'}
+    new_user_env = user_env(new_user['name'], new_user['password'],
+                            new_user['project'])
+
+    # Create role
+    role = openstack_client('role create {0} -f json'.format(role_name))
+    role = openstack_client.details(role)
+
+    # Save original file
+    cmd = "yes n | cp -i /etc/murano/policy.json /etc/murano/policy.json_orig"
```

Comment:
You can move patching file to `restore_murano_policy` fixture (before yield) and rename in to `patch_murano_policy` for example.